src/views/settings/SettingsSubjects.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState, useCallback, useLayoutEffect, useMemo } from "react";
-import { Alert, FlatList, KeyboardAvoidingView, Text, TextInput, TouchableOpacity, View } from "react-native";
+import { Alert, FlatList, KeyboardAvoidingView, Text, TextInput, TouchableOpacity, View, Modal, StyleSheet, Button } from "react-native";
```

Comment:
pour une indentation parfaite : 
```suggestion
import {
  Alert,
  FlatList,
  KeyboardAvoidingView,
  Text,
  TextInput,
  TouchableOpacity,
  View,
  Modal,
  StyleSheet,
  Button
} from "react-native";
```